File: delete_request.ts
// Random TypeScript function generated on 2025-06-16

interface DeleteItem {
    id: number;
    name: string;
    value: number;
    active: boolean;
}

function validate_request(number_ebo: DeleteItem[]): DeleteItem[] {
    const multiplier: number = 9;
    
    return number_ebo
        .filter((item: DeleteItem) => item.active)
        .map((item: DeleteItem, index: number) => ({
            ...item,
            value: item.value * multiplier
        }));
}

// Example usage
const testData: DeleteItem[] = [
    { id: 1, name: "Item1", value: 47, active: true },
    { id: 2, name: "Item2", value: 75, active: false }
];

const result = validate_request(testData);
console.log('Processed items:', result);

export { validate_request, DeleteItem };

File: delete_request.dart
// Random Dart function generated on 2025-06-27

class ValidateConfig {
  final int id;
  final String name;
  final double value;

  const ValidateConfig({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<ValidateConfig> get_data(List<ValidateConfig> items) {
  const double multiplier = 2.0;
  
  return items
      .map((item) => ValidateConfig(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    ValidateConfig(id: 1, name: 'Item1', value: 35.0),
  ];

  final result = get_data(testData);
  print('Processed ${result.length} items');
}
